cedar-policy/CHANGELOG.md

```diff
@@ -26,25 +26,30 @@ Cedar Language Version: 4.0
   around the entity type arguments. See [RFC 55](https://github.com/cedar-policy/rfcs/blob/main/text/0055-remove-unspecified.md).
 - Significantly reworked all public-facing error types to address some issues
   and improve consistency. See issue #745.
-- Finalized the `ffi` module which was preview-released in 3.2.0.
-  This involved a few additional API breaking changes in `ffi`. See #757.
+- Finalized the `ffi` and `wasm` modules which were preview-released in 3.2.0.
```

Comment:
Do we actually have a `wasm` module, or just a `cedar-wasm` crate?

Reply:
A `cedar-wasm` crate - updated text